numpy/core/numeric.py

```diff
@@ -2675,12 +2675,13 @@ def seterrcall(func):
         Function to call upon floating-point errors ('call'-mode) or
         object whose 'write' method is used to log such message ('log'-mode).
 
-        The call function takes two arguments. The first is the
-        type of error (one of "divide", "over", "under", or "invalid"),
+        The call function takes two arguments. The first is a string describing the
+        type of error (such as "divide by zero", "overflow", "underflow", or "invalid value"),
         and the second is the status flag.  The flag is a byte, whose
-        least-significant bits indicate the status::
+        least-significant bit indicates the error type (one of "divide", "over",
+        "under", "invalid")::
```

Comment:
I think the last part would be better as 'The flag is a byte whose four least-significant bits indicate the type of err, one of "divide", "over", "under", "invalid" '